rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -541,6 +541,11 @@ def element_attribute(element, name)
           execute :getElementAttribute, id: element.ref, name: name
         end
 
+        # Backwards compatibility for w3c
+        def element_property(element, name)
+          execute :getElementAttribute, id: element, name: name
```

Comment:
Right, Ruby implements W3C calls in a separate class. I don't think we should add methods to the JWP implemented Bridge to match new W3C calls.